boards/arm/adafruit_feather_stm32f405/pinmux.c

```diff
@@ -22,6 +22,16 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PB6, STM32F4_PINMUX_FUNC_PB6_I2C1_SCL},
 	{STM32_PIN_PB7, STM32F4_PINMUX_FUNC_PB7_I2C1_SDA},
 #endif /* CONFIG_I2C_1 */
+#ifdef CONFIG_SPI_1
+#ifdef CONFIG_SPI_STM32_USE_HW_SS
+	{STM32_PIN_PA15, STM32F4_PINMUX_FUNC_PA15_SPI1_NSS |
+			 STM32_OSPEEDR_VERY_HIGH_SPEED},
+#endif /* CONFIG_SPI_STM32_USE_HW_SS */
+	{STM32_PIN_PB3, STM32F4_PINMUX_FUNC_PB3_SPI1_SCK |
```

Comment:
text above in the `index.rst`  is mentioning "SPI1 uses SCK (PA5)", but PB3 is set as SCK here. 
is this mismatching ?

Reply:
Good catch, updated.